core/src/main/java/io/questdb/std/DirectLongList.java

```diff
@@ -73,31 +64,8 @@ public final void add(DirectLongList that) {
         this.pos += thatSize;
     }
 
-    public void sortAsUnsigned() {
-        Vect.sortULongAscInPlace(address, size());
-    }
-
     public long binarySearch(long v) {
-        long low = 0;
-        long high = ((pos - start) >> 3) - 1;
-
-        while (low <= high) {
-
-            if (high - low < 65) {
-                return scanSearch(v, low, high);
-            }
-
-            long mid = (low + high) >>> 1;
-            long midVal = Unsafe.getUnsafe().getLong(start + (mid << 3));
-
-            if (midVal < v)
-                low = mid + 1;
-            else if (midVal > v)
-                high = mid - 1;
-            else
-                return mid;
-        }
-        return -(low + 1);
+        return Vect.binarySearch64Bit(start, v, 0, (pos - start) / 8, BinarySearch.SCAN_UP);
```

Comment:
This was `binarySearchUniqueList` but now it isn't and the name does not tell anything about it. Branchless BS will perhaps perform better here.

Reply:
I thought to converge on some sort of naming convention, where `binarySeach` method name would be for non-unique BS and `binarySearchUniqueList` is for narrower unique lists. The reason for the change is to remove code duplication. This is an unused method outside of tests.